plugins/modules/web_infrastructure/jira.py

```diff
@@ -314,7 +314,14 @@ def request(url, user, passwd, timeout, data=None, method=None):
     if info['status'] not in (200, 201, 204):
         error = json.loads(info['body'])
         if error:
-            module.fail_json(msg=error['errorMessages'])
+            msg = ''
```

Comment:
Add changelog entry for this change.

Reply:
https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to
please, follow the format
```
bugfixes:
- jira - short-description (https://github.com/ansible-collections/community.general/pull/311).
```